lib/sshutils/signer.go

```diff
@@ -18,17 +18,57 @@ package sshutils
 
 import (
 	"crypto"
+	"io"
 
 	"golang.org/x/crypto/ssh"
 
-	"github.com/gravitational/teleport/api/utils"
-
 	"github.com/gravitational/trace"
 )
 
-// AlgSigner has been moved to /api/utils, and is now
-// imported here for backwards compatibility. DELETE IN 7.0.0
-var AlgSigner = utils.AlgSigner
+// AlgSigner wraps the provided ssh.Signer to ensure signature algorithm
```

Comment:
@Joerger @awly This is fine for now since `signer.go` contains similar code, but I think we can do some cleanup with the `sshca` and `sshutils` package.

Seems like we are just throwing things into these two packages at random at the moment. For example, this function (and related) seem like they should belong in `sshca`.

Reply:
This is where it was originally before my refactors, but I definitely agree we can organize it better.